test/unit_tests/utils/LineChartComparison_test.js

```diff
@@ -0,0 +1,94 @@
+/* global before describe it */
+
+'use strict';
+
+require( 'jsdom-global' )();
+
+const chai = require( 'chai' );
+const mock = require( 'mock-require' );
+const expect = chai.expect;
+const noop = () => {
+  // do nothing
+};
+
+mock( 'highcharts/js/highstock', {
+  setOptions: noop,
+  stockChart: ( props, opts ) => {
+    const options = opts;
+    return {
+      update: newOptions => Object.assign( options, newOptions ),
+      options: options,
+      showLoading: noop,
+      hideLoading: noop
+    };
+  }
+} );
+
+mock( 'highcharts/js/modules/accessibility', () => {
+  // do nothing
+} );
+
+const LineChart = require( '../../../src/js/charts/LineChartComparison.js' );
+let line;
+
+describe( 'LineChartComparison', () => {
+
+  before( () => {
+    line = new LineChart( {
+      el: 'el',
+      description: 'chart description!',
+      metadata: 'foo',
+      data: [
+        {
+          meta: {
+            name: 'foo'
+          },
+          data: [ {
+            date: 1199163600000,
+            foo: 1,
+            bar: 2
```

Comment:
consider using fake data with real meaning to give context to what these tests are testing for. good example of alternative to foobar: https://gist.github.com/piotrlewandowski/2cdaaa0edc1423e996c5